packages/core/src/ui/components/WalletSetupRevamp/WalletSetupMnemonicStepRevamp/WalletSetupMnemonicStepRevamp.tsx

```diff
@@ -109,9 +108,9 @@ export const WalletSetupMnemonicStepRevamp = ({
         >
           {translations.writePassphraseSubtitle2}
         </span>
-        <Dialog.Root open={videoModalOpen} setOpen={setVideoModalOpen}>
+        <Modal open={videoModalOpen} centered footer={undefined} closable={false} destroyOnClose maskClosable={false}>
```

Comment:
`Dialog.Root` has a `zIndex` prop. Modal from antd applies a `z-index` of 1000 so we need to set it to a higher number.
```suggestion
        <Dialog.Root open={videoModalOpen} setOpen={setVideoModalOpen} zIndex={1001}>
```
I don;t like that it is being passed in the markup but this is how it is implemented now :/. Despite that I would still prefer to use lace ui as the goal is to introduce it gradually and eliminate antd at some point.

We could also change the root modal of the multiwallet to lace-ui's one and that would also eliminate a problem without setting the zIndex, but I don't request it here 😉

Reply:
Resolved